tests/src/lib.rs

```diff
@@ -95,6 +95,7 @@ mod utils {
     }
 
     #[derive(Debug)]
+    #[allow(dead_code)]
     pub enum FileOps {
         Python,
         Rust,
```

Comment:
I think the thing here is that `Python` and `Rust` are only used on linux, so alternatively to allowing dead code, we could also conditionally compile them only on linux. Just an idea, up to you.